### DISCUSSION TITLE
Shinny Apps

### DISCUSSION THREAD

**Discussion by Anggitnoviee** [teamalgoritma/community#223] [Q&A] [answered]
Jadi saya mau membuat dashboard di shinny apps, dan menggunakan data rahasia perusahaan, apakah shinny apps bisa mempunyai atau bisa dibuat tempat login? seperti mempunyai userid dan password?? Terimakasih

**Comment by ysitta** [comment, marked as answer]
Dear @Anggitnoviee,

Terima kasih atas pertanyaannya. Terkait login page di shiny, Ibu bisa memanfaatkan shinymanager. Sehingga, dibutuhkan untuk input username dan password terlebih dahulu sebelum masuk ke page utama di shiny Ibu. Terlampir contoh penggunaan dan codenya ya, Bu
https://stackoverflow.com/questions/28987622/starting-shiny-app-after-password-input. 

Demikian yang dapat kami sampaikan. Semoga membantu :)

Apabila sekiranya jawaban ini membantu, mohon ditandai sebagai answered ya bu :)
